### ISSUE TITLE
Get-ComputerInfo and OsLocalDateTime

### ISSUE THREAD

**Issue by iSazonov** [PowerShell/PowerShell#3146]
~~~Currently Get-ComputerInfo return empty OsLocalDateTime.~~~

Also we need fix test for OsLocalDateTime in Microsoft.PowerShell.Management/Get-ComputerInfo.Tests.ps1 ( It "(special case) Test for property = OsLocalDateTime" ) because it do nothing.

**Comment by jeffbi** [comment]
Odd, it works for me.
```
PS > Get-Computerinfo OsLocalDateTime

OsLocalDateTime
---------------
2/14/2017 1:43:42 PM

PS > Get-Computerinfo
  ...
OsLocaleID              : 0409
OsLocale                : en-US
OsLocalDateTime         : 2/14/2017 1:45:57 PM
OsLastBootUpTime        : 2/1/2017 6:02:34 PM
OsUptime                : 12.19:43:22.8442170
  ...
```

As for the test, there are a number of tests in which we cannot reasonably compare one result against another since the results are not fixed, such as `OsFreePhysicalMemory` and `OsLocalDateTime`. What we could do is test that they return non-null values of the appropriate type. All result properties are nullable types.

**Comment by iSazonov** [comment]
Do it makes sense to make a comparison with the current date?

**Comment by jeffbi** [comment]
The test could store the current time before and after executing `Get-ComputerInfo`, then check that the `OsLocalTime` value falls between those two times. The `OsLocalTime` value should be very close to the time the cmdlet was started because the `Win32_OperatingSystem` information is gathered first.

**Comment by kalgiz** [comment]
Fixed in #6253
